### PR TITLE
Story2 add item

### PR DESCRIPTION
**Pull request by Albertoroldanq**
I think the website meets the requirements, but I am not sure if the code correctly done

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -9,21 +10,56 @@
 $query->execute();
 $allWines = $query->fetchAll();
 
+$addWineQuery = $db->prepare('INSERT INTO `list-of-wines` (`name`,`origin`, `type`, `grape`) VALUES (:name, :origin, :type, :grape );');
+$addWineQuery->bindParam(':name', $nameOfWine);
+$addWineQuery->bindParam(':origin', $typeOfWine);
+$addWineQuery->bindParam(':type', $countryOfWine);
+$addWineQuery->bindParam(':grape', $grapeOfWine);
+
+if(count($_SESSION) && isset($_POST)) {
```
Comment (norayna) [inline index.php:19]: the check `count($_SESSION)` is not really working - i still get this error when opening the page for the first time:
image

hint: check that the session has the right number of things in it

**Review by norayna — CHANGES_REQUESTED** [review]
only small changes requested

File: index.php (hunk 1 above)
Comment (Albertoroldanq) [reply index.php:19]: Thanks! Let's see if it works now...

File: style.css [hunk 2]
```diff
@@ -1,32 +1,52 @@
...
+body{
+    width:100%;
+}
 
 * {
+    font-family: 'Montserrat', sans-serif;
     box-sizing: border-box;
 }
 
 h1{
```
[the remark this answers is not in the document]
Comment (Albertoroldanq) [reply style.css:17]: I will update it in a new task :)

**Review by norayna — APPROVED** [review]

**Pull request merged**